src/components/Error/ErrorMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import CloseButton from '../Button/CloseButton';
+
+interface Props {
+  errorMessage: string;
+  onClose?: () => void;
+}
+
+const ErrorMessage: React.FC<Props> = ({ errorMessage, onClose }) => {
+  return (
+    <ErrorBanner>
+      <Error>{errorMessage ?? errorMessage}</Error>
+      <CloseButton onClick={onClose} top={0} right={0} />
+    </ErrorBanner>
+  );
+};
+
+const ErrorBanner = styled.div`
+  position: fixed;
```

Comment:
it should stay within component boundaries and not leave outside because component should not affect dapp interface

Reply:
also just noticed you are setting it as `relative` below, make sure u fix this